### ISSUE TITLE
Gnome 46 Support

### ISSUE THREAD

**Issue by dik23** [OttoAllmendinger/gnome-shell-screenshot#206]
Gnome 46 is now live, please add support

Thanks!

**Comment by Eeems** [comment]
It looks like it won't just work by bumping the shell-version : `this.panelButton.add_actor is not a function`

**Comment by andymai** [comment]
Support for Gnome 46 has been merged! https://github.com/OttoAllmendinger/gnome-shell-screenshot/pull/208

**Comment by Haelle** [comment]
could we have a new release then ? 🙏

**Comment by tuandev1l** [comment]
well, i've been here for 9 months to receive new release.

**Comment by msjtheriault** [comment]
@OttoAllmendinger Can we get a release please?

**Comment by pacho2** [comment]
Could the version in extensions.gnome.org be updated please? Thanks a lot

**Comment by rabyj** [comment]
It doesn't seem to work on my end (Gnome Shell 46, Ubuntu 24.04)

Journal shows this error if I try to capture something:
```
May 28 18:27:36 computerName gnome-shell[10934]: Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
                                                    wrapNotifyError/<@file://$HOME/.local/share/gnome-shell/extensions/gnome-shell-screenshot@ttll.de/extension.js:1172:27
                                                    activate@resource:///org/gnome/shell/ui/popupMenu.js:195:14
                                                    _init/<@resource:///org/gnome/shell/ui/popupMenu.js:112:24
                                                    @resource:///org/gnome/shell/ui/init.js:21:20
```

**Comment by Cookies77777** [comment]
It does not work for me either on Ubuntu 24.04LTS, same error as rabyj:
```
Jun 12 10:47:07 cookies-xe gnome-shell[3384]: Unhandled promise rejection. To suppress this warning, add an error handler to your promise chain with .catch() or a try-catch block around your await expression. Stack trace of the failed promise:
                                                wrapNotifyError/<@file:///home/cookies/.local/share/gnome-shell/extensions/gnome-shell-screenshot@ttll.de/extension.js:1172:27
                                                activate@resource:///org/gnome/shell/ui/popupMenu.js:195:14
                                                _init/<@resource:///org/gnome/shell/ui/popupMenu.js:112:24
                                                @resource:///org/gnome/shell/ui/init.js:21:20
```
If I select Desktop Portal in back-end settings then it does take the screenshot but its more clicks and the selected area is remembered from last screenshot and that's super annoying.

**Comment by pacho2** [comment]
Do you have gnome-screenshot installed on your systems? I think I saw it being dropped on some Ubuntu systems.

In my case I am using it on Gentoo with Gnome up to 47 and it works fine with gnome-screenshot backend.

**Comment by Cookies77777** [comment]
> Do you have gnome-screenshot installed on your systems? I think I saw it being dropped on some Ubuntu systems.
> 
> In my case I am using it on Gentoo with Gnome up to 47 and it works fine with gnome-screenshot backend.

Yes, they removed gnome-screenshot but I installed in manually. I double checked now and it's installed.
If I select gnome-screenshot backend in the extension then I can select the screenshot area and the area flashes white but that's it. There is no pop notification that it was successful and that error above shows up in the journalctl.


Also after messing around more, I found that the issue is actually the popup notification itself.
I just disabled the  "Show Notification After Capture" option in settings and now it takes screenshots fine using gnome-screenshot backend.
It would be nice to have the notification but I can live without it.
I'm also using the "Notification Banner Reloaded" Extension to move the notifications to bottom right, maybe there's a conflict between the extensions but all other notifications seem to work fine.

**Comment by rabyj** [comment]
@Cookies77777 I found that you can set the program to Auto-Copy to clipboard the image data, if that helps with anything. It was what was missing from me. I used to use the notification to copy to clipboard.
